docs/guides/js-waku/quick-start.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Quick Start
+---
+
+This guide provides quick steps to start using the `js-waku` SDK by setting up a Waku node and sending messages using the [Relay protocol](/overview/concepts/protocols#relay). Please refer to the [installation guide](/guides/js-waku/#installation) for steps on adding `js-waku` to your project.
+
+## Create a Relay Node
+
+Use the `createRelayNode()` function to create a relay node and interact with the Waku Network:
+
+```js
+import { createRelayNode } from "@waku/sdk";
+
+// Create and start a relay node
+const node = await createRelayNode({ defaultBootstrap: true });
+await node.start(); // Use the stop() function to stop a running node
+```
+
+:::info
+The `defaultBootstrap` option bootstraps your node using pre-defined Waku nodes.
```

Comment:
Maybe in order not to duplicate this comment across docs - I can update the appropriate interface with some description comment and we can just refer to it in the docs.

Reply:
I only mentioned this info once, so there's no duplicate to worry about. I still support updating the interface as it'll be helpful for users exploring the [API Documentation](https://js.waku.org/).